packages/client/src/debounce.tsx

```diff
@@ -0,0 +1,18 @@
+export default function debounce(
+  func: Function,
```

Comment:
The type for `func` should be more precise. In particular `debounce` should return a function with the same signature as `func`. That is something we can work on together.